### PR TITLE
feat(sqlserver): add SqlServer database image

### PR DESCRIPTION
**Pull request by educhastenier**
Relates to [CI-567](https://bonitasoft.atlassian.net/browse/CI-567)

### CODE REVIEW

File: sqlserver/2017/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
+FROM mcr.microsoft.com/mssql/server:2017-CU22-ubuntu-16.04
+
+LABEL maintainer="Bonita R&D Runtime team <rd.runtime@bonitasoft.com>"
```
Comment (baptistemesta) [inline sqlserver/2017/Dockerfile]: Should that be public?

Comment (baptistemesta) [reply sqlserver/2017/Dockerfile]: Maybe put the list of dev in the runtime team?

File: sqlserver/2019/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,25 @@
+FROM mcr.microsoft.com/mssql/server:2019-CU8-ubuntu-16.04
+
+LABEL maintainer="Bonita R&D Runtime team <rd.runtime@bonitasoft.com>"
```
Comment (baptistemesta) [inline sqlserver/2019/Dockerfile]: same remark on maintainers

**Review by baptistemesta — APPROVED** [review]

**Comment by baptistemesta** [comment]
LGTM, However, I'm not sure what we should put as maintainers. that email address should not be used outside of Bonitasoft. I'm nitpicking a little.

File: sqlserver/2017/Dockerfile (hunk 1 above)
Comment (educhastenier) [reply sqlserver/2017/Dockerfile]: why, this is more generic like this if someone leaves or comes into the team

Comment (educhastenier) [reply sqlserver/2017/Dockerfile]: After checking, we cannot access this address from the outside world, so I'll do as you suggest.

Comment (educhastenier) [reply sqlserver/2017/Dockerfile]: fixed

**Comment by baptistemesta** [comment]
@educhastenier I don't have the right to merge PR, but it looks good to me.

**Pull request merged**